pylint/extensions/magic_value.py

```diff
@@ -0,0 +1,89 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+"""Checks for magic values instead of literals."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from astroid import nodes
+
+from pylint.checkers import BaseChecker, utils
+from pylint.checkers.base.comparison_checker import ComparisonChecker
```

Comment:
Can't these be utils?

Reply:
Why are we importing a checker here? Can't the method we need from the checker be an utility function in `util.py`? Creating such coupling between checkers is generally something we avoid.

Reply:
I understand now you are right :)